admin_guide/encrypting_data.adoc

```diff
@@ -286,5 +298,10 @@ resources:
           secret: <BASE 64 ENCODED SECRET>
 ----
 
-and restart all master processes. Then run the command `oc adm migrate storage --include=secrets --confirm`
-to force all secrets to be decrypted.
+. Restart all master processes.
```

Comment:
@php-coder Is this the correct command for this step? I think we should show it:
# systemctl restart atomic-openshift-master

Reply:
It turned out that restarting API server(s) should be enough in this case:
```
<semushin> claytonc, in the doc about encryption at rest once you suggest to restart "all master processes" and in another place "restart API server". Why do we need to restart a full master? Sound like restarting API server should be sufficient in all the cases.
<claytonc> semushin: yes apiserver is minimal
<claytonc> that's what should be restarted
<claytonc> for secrets
```

Reply:
@php-coder So remove all mentions of restarting master processes across the page, or just this section?

Reply:
Not removing but replacing:
```diff
-. Restart all master processes.
+. Restart all API servers.
```